pom.xml

```diff
@@ -95,14 +95,14 @@
 
   <distributionManagement>
     <snapshotRepository>
-      <id>sonatype-nexus-snapshots</id>
-      <name>Sonatype Nexus Snapshots</name>
-      <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
+      <id>njord-snapshots</id>
+      <name>Local Snapshots Staging</name>
+      <url>njord:snapshot-sca</url>
     </snapshotRepository>
     <repository>
-      <id>sonatype-nexus-staging</id>
-      <name>Nexus Release Repository</name>
-      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+      <id>njord-releases</id>
```

Comment:
The project.distributionManagement.repository.id is unused. What if we set this to `sonatype-cp` (which does corresponds to reality) and Njord use this info to default publisher (and auth for it, based on server.id?)

Reply:
Need to tinker more: https://the-asf.slack.com/archives/C7Q9JB404/p1746012791622819?thread_ts=1746011932.174119&cid=C7Q9JB404